notebooks/VectorSearch_QuestionRetrieval.ipynb

```diff
@@ -0,0 +1,254 @@
+{
```

Comment:
<p>Nitpick, can we use back-ticks around <code>pylibraft</code>?  <span class="ql-cursor">﻿</span></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/raft/pull/1662/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>